common/src/main/scala/explore/components/Tile.scala

```diff
@@ -54,7 +57,7 @@ object Tile {
   def renderPortal(mountNode: Option[raw.ReactDOM.Container], info: VdomNode): VdomNode =
     mountNode.map(node => ReactPortal(info, node))
 
-  implicit val rawContainerReuse: Reusability[raw.ReactDOM.Container] = Reusability.always
+  implicit val rawContainerReuse: Reusability[raw.ReactDOM.Container] = Reusability.never
```

Comment:
@rpiaggio, why was this `always`?

Reply:
The DOM `Element` (`<span>`)  where the portal is mounted will always be the same during the lifetime of the `Tile`. Therefore, by setting this to `always` we ensure that the same `ReactPortal` is used if the `Tile` rerenders for any reason, otherwise it will be recreated.

Was this preventing necessary rerenders?

Reply:
Actually I didn't see any difference, However the fulll Tile is set to always re render because we don't really know what's inside the body, though perhaps it can be done via `Reuse`

Reply:
BTW, now that we have reusable functions maybe we want to derive `propsReuse`? Or use a `.by(...)` so we can take the `render` function into account too.

Reply:
If we change reuse to be smarter, I believe we should still keep `rawContainerReuse` as `never`.

Reply:
Ok, I'll do that on another PR, to learn a bit about `Reuse`